test/repository/hbase/enterprise/HBaseRestEnterpriseUnitRepositorySpec.scala

```diff
@@ -0,0 +1,73 @@
+package repository.hbase.enterprise
+
+import java.time.Month.SEPTEMBER
+
+import scala.concurrent.Future
+
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{ EitherValues, FreeSpec, Matchers }
+
+import uk.gov.ons.sbr.models.Period
+import uk.gov.ons.sbr.models.enterprise.Enterprise
+
+import repository.RestRepository.Row
+import repository.hbase.HBase.DefaultColumnFamily
+import repository.{ RestRepository, RowMapper }
+import support.sample.SampleEnterpriseUnit
+
+class HBaseRestEnterpriseUnitRepositorySpec extends FreeSpec with Matchers with MockFactory with ScalaFutures with EitherValues {
+
+  private trait Fixture extends SampleEnterpriseUnit {
+    val TargetTable = "enterprise"
```

Comment:
Method, Value and variable names should be in lower camel case

Reply:
Again, this is a constant and adheres to the Scala convention.

Reply:
Message received and understood